com.unity.formats.fbx/Editor/Unity.Formats.Fbx.Editor.asmdef

```diff
@@ -20,7 +20,7 @@
     "versionDefines": [
         {
             "name": "com.unity.recorder",
-            "expression": "2.2.0-preview.4",
+            "expression": "2.5.5",
```

Comment:
please update the changelog and docs (if applicable) that we are updating the recorder requirement

Reply:
Done!